go.mod

```diff
@@ -1,13 +1,86 @@
 module gitlab.com/rackn/terraform-provider-drpv4
 
-go 1.13
+go 1.18
 
 require (
-	github.com/Bowery/prompt v0.0.0-20190916142128-fa8279994f75 // indirect
-	github.com/VictorLowther/jsonpatch2 v1.0.0
-	github.com/go-test/deep v1.0.3
-	github.com/hashicorp/terraform v0.13.2
-	github.com/hashicorp/terraform-plugin-sdk/v2 v2.0.1
-	github.com/pborman/uuid v1.2.0
-	gitlab.com/rackn/provision/v4 v4.8.0-alpha00.0.20210905035228-7afe04a243cf
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.11.1
```

Comment:
I don't think this is the latest, would it be possible to update?

Reply:
I was able to update to v2.24.1. With v2.25.0 it breaks DRP provision library as it uses some old version of a Hashicorp library that has breaking change.